Ontime.js

```diff
@@ -94,7 +100,6 @@ function setEventData(eventObject, payload) {
     eventObject.public.set(false);
     eventObject.skip.set(false);
     eventObject.note.set('');
-    //TODO: Colour conversion
```

Comment:
Let's keep this so we don't forget

Reply:
Only cleared it for the situation where null is recieved, the TODO is still there in the payload section.